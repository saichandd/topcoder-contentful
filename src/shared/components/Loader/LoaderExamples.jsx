import React from 'react';
import Loader from './Loader';

const LoaderExamples = () => (
  <div className="LoaderExamples flex column middle center">
    <h1>
Default
    </h1>

    <Loader />
  </div>
);

module.exports = LoaderExamples;
