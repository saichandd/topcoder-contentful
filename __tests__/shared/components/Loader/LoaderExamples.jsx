import React from 'react';
import Renderer from 'react-test-renderer/shallow';
import LoaderExamples from 'components/Loader/LoaderExamples';

test('Matches shallow shapshot', () => {
  const renderer = new Renderer();
  renderer.render((
    <LoaderExamples />
  ));
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
