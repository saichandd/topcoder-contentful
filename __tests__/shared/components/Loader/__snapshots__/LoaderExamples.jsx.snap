// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow shapshot 1`] = `
<div
  className="LoaderExamples flex column middle center"
>
  <h1>
    Default
  </h1>
  <Loader
    type=""
  />
</div>
`;
